egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -270,7 +270,8 @@ def AddLstmLayer(config_lines,
                  ng_per_element_scale_options = "",
                  ng_affine_options = "",
                  lstm_delay = -1,
-                 self_repair_scale = None):
+                 self_repair_scale = None,
```

Comment:
@freewym please add comments explaining the differences between `self_repair_scale` and `self_repair_scale_clipgradient` in all the methods in `steps/nnet3/components.py` which use these features.